base/client.jl

```diff
@@ -71,6 +72,19 @@ function repl_cmd(cmd, out)
         ENV["OLDPWD"] = new_oldpwd
         println(out, pwd())
     else
+        @windows_only begin
+            if lowercase(shell[1]) == "cmd"
+                cmdbuf = IOBuffer()
+                print_cmdshell_escaped(cmdbuf, cmd.exec...)
+                cmd.exec = ["cmd"; "/s /c \"$(bytestring(cmdbuf.data))\""]
+                cmd.verbatimargument = true
+            elseif lowercase(shell[1]) == "powershell"
+                if length(cmd.exec) >= 2 && cmd.exec[2] == "--%"
+                  cmd.exec[2] = "\"--%\""
```

Comment:
BTW, when using `powershell -command`, one also needs to quote `"` as `\"` in addition to the `` ` `` stuff.

http://www.tinyint.com/index.php/2011/04/20/escaping-quotes-in-powershell-exe-command-via-command-prompt/